tests/query/processors/test_mandatory_condition_applier.py

```diff
@@ -0,0 +1,134 @@
+import copy
+from typing import List
+
+import pytest
+
+from snuba.datasets.schemas import MandatoryCondition
+from snuba.datasets.schemas.tables import TableSource
+from snuba.query.conditions import (
+    OPERATOR_TO_FUNCTION,
+    BooleanFunctions,
+    ConditionFunctions,
+    binary_condition,
+    combine_and_conditions,
+)
+from snuba.query.expressions import Column, Literal
+from snuba.query.logical import Query
+from snuba.query.processors.mandatory_condition_applier import MandatoryConditionApplier
+from snuba.request.request_settings import HTTPRequestSettings
+
+test_data = [
+    (
```

Comment:
Please use pytest.param so that you can assign these cases a name
```
pytest.param(
  "table1",
   [..your mandatory conditions ...],
   ... more params if you have any ...
   id="Name of the test case"
```

The id will be shown in the pytest report so we know which case fails when they fail.